src/main/drivers/timer.h

```diff
@@ -210,3 +210,5 @@ uint16_t timerDmaSource(uint8_t channel);
 uint16_t timerGetPrescalerByDesiredHertz(TIM_TypeDef *tim, uint32_t hz);
 uint16_t timerGetPrescalerByDesiredMhz(TIM_TypeDef *tim, uint16_t mhz);
 uint16_t timerGetPeriodByPrescaler(TIM_TypeDef *tim, uint16_t prescaler, uint32_t hz);
+
+int8_t timerGetNumber(const TIM_TypeDef *tim);
```

Comment:
maybe `timerGetTIMNumber`, but there should be even better name. `Number` sounds quite ambiguous in this context ...

Reply:
There is a separate PR for this one. I will fix it there first.